data-raw/create_queries_and_queries_mh.R

```diff
@@ -5,6 +5,9 @@
 # SRCVAR, e.g., AEDECOD, AELLT, ...
 # TERMCHAR, non NULL
 
+library(dplyr) # nolint
+library(admiral) # nolint
```

Comment:
Thinking on this - is this needed with the use of `::` and `:::`?

Reply:
You are right, we don't need to load admiral. I've removed it. Thanks.